recipe/meta.yaml

```
@@ -250,9 +253,8 @@ outputs:
       run:
         - {{ pin_compatible("cuda-version", min_pin="x", max_pin="x") }}
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
-        - cudatoolkit       # [(cuda_compiler_version or "").startswith("11")]
-        - cuda-nvrtc        # [(cuda_compiler_version or "").startswith("12")]
-        - libcublas         # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-nvrtc        # [(cuda_compiler_version or "").startswith(("12", "13"))]
+        - libcublas         # [(cuda_compiler_version or "").startswith(("12", "13"))]
```

We can drop the selectors here as this is now true of all CUDA versions. This will help simplify our maintenance going forward

```suggestion
        - cuda-nvrtc
        - libcublas
```

---

Done, thanks! I made some additional changes while at it.